tests/FSharp.Configuration.Tests/FSharp.Configuration.Tests.fsproj

```diff
@@ -2,12 +2,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <!-- If you want to test multiple, you need switch in build.fsx: -->
+    <!--<TargetFrameworks>net48;net8.0</TargetFrameworks>-->
```

Comment:
do you still want to build this for net48?

Reply:
This TP still works well with .NET Framework, and pre-2.0 versions of this TP are .NET Framework only. If you look at the [Nuget stats of this repo](https://www.nuget.org/stats/packages/FSharp.Configuration?groupby=Version), the last 6 weeks have still thousands of v1.5 downloads. Ultimately, we would like them to be able to update to FSharp.Configuration 3, even if they stay with .NET Framework, because it makes their future migration to dotnet easier for them.

In that sense, it wouldn't hurt to have the same unit tests running in both target frameworks.

Reply:
so, your idea is to ship `netstandard20` binaries for `net48`, tests for `net8.0` but have an option to manually test `net4.8` on windows machines?

Reply:
I think currently the TPs make sense to have both runtime and design-time in netstandard2.0+net8.0. The reason being that if there are reference assemblies used by the runtime (like System.Configuration.ConfigurationManager in this case), you can see that they target to net8.0 separately. Then if TP runtime targets only to netstandard2.0 then it has to use the netstandard2.0 version of the dependency, right? Which could lack some performance work that is included in their net8.0 version.  BTW, also same for SwaggerProvider, which has now runtime only in netstandard2.0.